wavefront/config/config.pb.go

```diff
@@ -1,39 +1,29 @@
 // Code generated by protoc-gen-gogo. DO NOT EDIT.
```

Comment:
It looks like nothing has changed in `config.proto`. I was wondering why this file was updated.